MailCore/Cache/AccountManager.swift

```diff
@@ -304,6 +304,10 @@ public class AccountManager: RefreshTokenDelegate {
         setCurrentMailboxForCurrentAccount(mailbox: mainMailbox)
         saveAccounts()
 
+        Task {
+            try await currentContactManager?.fetchContactsAndAddressBooks()
```

Comment:
We must fetch contacts and adress books when we create a new contact. Could you do it?